.github/workflows/dotnetcore-api.yml

```diff
@@ -1,5 +1,5 @@
-# build and test PayrollProcessor.Api
-name: dotnet core PayrollProcessor.Api - build
+# build and test PayrollProcessor.Web.Api
+name: dotnet core PayrollProcessor.Web.Api - build
```

Comment:
Maybe we want to simplify the CI build to just build the solution since all .NET code is contained in it now?